composer-templates/template.composer.json

```diff
@@ -39,6 +39,9 @@
       }
     },
     "drupal-scaffold": {
+      "locations": {
```

Comment:
Can we add the following section so that things like the .htaccess and robots.txt are not overwritten whenever Drupal core is updated:

```
"file-mapping": {
    "[web-root]/.htaccess": false,
    "[web-root]/robots.txt": false
}
```

Reply:
Added.